NetSpell.SpellChecker/Dictionary/WordDictionary.cs

```diff
@@ -14,8 +14,8 @@
 
 namespace NetSpell.SpellChecker.Dictionary
 {
-    using AffixRuleCollection = Dictionary<string, AffixRule>;
-    using PhoneticRuleCollection = List<PhoneticRule>;
+    using AffixRuleCollection = System.Collections.Generic.Dictionary<string, NetSpell.SpellChecker.Dictionary.Affix.AffixRule>;
+    using PhoneticRuleCollection = System.Collections.Generic.List<NetSpell.SpellChecker.Dictionary.Phonetic.PhoneticRule>;
```

Comment:
Why have these changed?

Reply:
A probable bug in the analyzer, but seemed to have low impact overall.

Reply:
Please revert. I can live with the rest.

Reply:
Done